development/ts-migration-dashboard/files-to-convert.json

```diff
@@ -10,8 +10,6 @@
   "app/scripts/controllers/app-state.js",
   "app/scripts/controllers/cached-balances.js",
   "app/scripts/controllers/cached-balances.test.js",
-  "app/scripts/controllers/detect-tokens.js",
-  "app/scripts/controllers/detect-tokens.test.js",
```

Comment:
These are no longer conversion targets as `TokenDetectionController.ts` and its test file are fully written in TypeScript.